### PR TITLE
Tests employee #5

### PR DESCRIPTION
**Pull request by paloma-ribeiro**
Esta PR contém:
- Refatoração do código
- Conexão com o banco de dados
- Refatoração dos testes do service e controller e criação dos testes do repository

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
+}
```
Comment (jcmalves) [inline .vscode/settings.json:3]: Was this folder created automatically? if so, could you deleted it for me?

File: cmd/server/routes/employeesRouter.go [hunk 2]
```diff
@@ -1,29 +1,25 @@
 package routes
```
Comment (jcmalves) [inline cmd/server/routes/employeesRouter.go:1]: In the last class, Cyro told us that `snack_case` names are commonly used for files and folder. Do you think that would be interesting to change `employeesRouter` to `employees_router`?

**Review by jcmalves — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (paloma-ribeiro) [reply .vscode/settings.json:3]: For some reason even added in .gitignore this folder still appears.
Folder deleted.

File: cmd/server/routes/employeesRouter.go (hunk 2 above)
Comment (paloma-ribeiro) [reply cmd/server/routes/employeesRouter.go:1]: Done.

**Pull request merged**